private-templates-service/client/src/App.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import { Container } from "reactstrap";
+
+import { UserAgentApplication, ClientAuthError } from "msal";
+
+import NavBar from "./components/NavBar/NavBar";
+import ErrorMessage from "./components/ErrorMessage/ErrorMessage";
+import Welcome from "./components/Welcome/Welcome";
+import config from "./Config";
+import { getUserDetails, getOrgDetails } from "./Services/GraphService";
+import { ErrorMessageProps } from "./components/ErrorMessage/ErrorMessage";
+
+import "bootstrap/dist/css/bootstrap.css";
+
+export interface UserType {
+  displayName?: string;
+  email?: string;
+  organization?: string;
+  avatar?: string;
+}
+
+// TODO: use react-redux
+interface State {
+  isAuthenticated: boolean;
+  user: UserType;
+  error: ErrorMessageProps | null;
+}
+
+class App extends Component<{}, State> {
+  userAgentApplication: UserAgentApplication;
+
+  constructor(props: {}) {
+    super(props);
+
+    console.log(JSON.stringify(props));
+
+    this.userAgentApplication = new UserAgentApplication({
+      auth: {
+        clientId: config.appId,
+        redirectUri: config.redirectUri
+      },
+      cache: {
+        cacheLocation: "localStorage",
+        storeAuthStateInCookie: true
+      }
+    });
+
+    var user = this.userAgentApplication.getAccount();
```

Comment:
In typescript we normally use `let` instead of `var` as let is block scoped and var isnt: https://www.tutorialsteacher.com/typescript/typescript-variable